config/webpack.config.prod.js

```diff
@@ -1,49 +1,13 @@
 const CopyPlugin = require('copy-webpack-plugin');
-const HtmlWebpackPlugin = require('html-webpack-plugin');
-const FaviconsWebpackPlugin = require('favicons-webpack-plugin');
-const Dotenv = require('dotenv-webpack');
-const path = require('path');
+const webpackMerge = require('webpack-merge');
+const common = require('./webpack.config.common');
 
-module.exports = {
+module.exports = webpackMerge(common, {
     mode: 'production',
```

Comment:
If you write the mode here you don't need the `--mode production` in the script on package.json